packages/contracts-bedrock/src/deployment/SystemDictator.sol

```diff
@@ -332,7 +333,7 @@ contract SystemDictator is OwnableUpgradeable {
         (bool success,) = address(config.implementationAddressConfig.addressDeprecatorImpl).delegatecall(
             abi.encodeWithSelector(bytes4(keccak256("deprecateAddresses()")))
         );
-        require(success, "Reverted: Error deprecating addresses");
```

Comment:
I haven't looked at the blame, but it's unclear to me why these messages are  changing?   There are a few other similar instances as well.

Reply:
I simplified the error message to reduce the contract size

Reply:
Are you concerned about this generating merge conflicts? Was the contract size becoming problematic?

Reply:
Yes, this can be the problem. Once we finish the migration, this contract can be removed.